design/20210715.bundle-crd.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Bundle CRD
+authors:
+  - "@joshvanl"
+editor: "@joshvanl"
+creation-date: 2021-07-15
+last-updated: 2021-07-15
+status: implemented
+---
+
+# Bundle CRD
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Goals](#goals)
+  * [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  * [Bundle](#bundle)
+    + [API](#api)
+    + [Custom Cache](#custom-cache)
+    + [Upstream CertificateBundle](#upstream-certificatebundle)
+  * [Testing](#testing)
+  * [Risks and Mitigations](#risks-and-mitigations)
+  * [Graduation](#graduation)
+      - [Alpha](#alpha)
+      - [Beta](#beta)
+      - [GA](#ga)
+<!-- /toc -->
+
+
+## Summary
+
+[Bundle](https://github.com/cert-manager/trust/blob/44d8fcaeec7232daf6dd14a1be5a6d0791bc23b8/pkg/apis/trust/v1alpha1/types_bundle.go)
+is a cluster scoped CRD designed to distribute trust data across a cluster so
+that it is made available by all services that need it. Bundle is intended to be
+complimentary to [cert-manager](https://github.com/jetstack/cert-manager) by
+allowing access to trust data that is both known to cert-manager, and external
+to any cert-manager [Issuer](https://cert-manager.io/docs/configuration/).
```

Comment:
nitpick: I think this reads like trust data needs to be both known and external to cert-manager

```suggestion
allowing access to trust data that is either known to cert-manager or external
to any cert-manager [Issuer](https://cert-manager.io/docs/configuration/).
```